pkg/backup/s3/s3.go

```diff
@@ -61,7 +65,7 @@ func New(bucket, prefix string, option session.Options) (*S3, error) {
 func (s *S3) Put(key string, rs io.ReadSeeker) error {
 	_, err := s.client.PutObject(&s3.PutObjectInput{
 		Bucket: aws.String(s.bucket),
-		Key:    aws.String(s.prefix + key),
+		Key:    aws.String(path.Join(v1, s.prefix, key)),
```

Comment:
another option is to prepend "v1" to "prefix" field. That should be easier. There is one special case in CopyPrefix() to handle.